couchbaselabs/sync-gateway-testcluster
# Multiple db test scenarios

Test scenarios
- Multiple dbs with shared buckets
- Mulitple dbs with unique buckets (first-pass, simple scenario)

Framework enhancements
- Rename functional_tests/cluster_setup -> functional_tests/fixtures
- Add expected number of docs to my verification function
- Add admin.register_bulk_users()
- Add server.delete_buckets()
- Add server.create_buckets(list_of_bucket_names)
- Add create-server-buckets.yml
- cluster.Reset(config) now functions in the below way:
   - Ask server what buckets exist
   - Delete those buckets
   - Parse the config to discover buckets that need to be created
   - Create buckets
   - reset-sync-gateway.yml
- reset-sync-gateway.yml no longer flushes buckets
   - stops sync_gateway
   - deploys config
   - starts sync_gateway
